### PR TITLE
Jon/aave/wyre-loan-create

### PR DESCRIPTION
**Pull request by Jon-edge**
This PR integrates the Wyre withdraw to bank option for the Loan Create workflow. Supports a withdrawal from BTC to Fiat

#### PR Dependencies

 none

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202974109503110

### CODE REVIEW

File: src/controllers/action-queue/WyreClient.js [hunk 1]
```diff
@@ -41,6 +42,8 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
     Accept: 'application/json'
   }
 
+  logyv(wyreSecret, 'wyreSecret')
```
Comment (samholmes) [inline src/controllers/action-queue/WyreClient.js:45]: This looks like it crept in.

File: src/controllers/action-queue/WyreClient.js [hunk 2]
```diff
@@ -83,10 +87,10 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
         if (blockchains == null) return
 
         // Special cases for testnets, if they don't yet support (they currently do not)
-        if (blockchains.MUMBAI == null) blockchains.MUMBAI = blockchains.MATIC
+        if (blockchains.MUMBAI == null && blockchains.MATIC != null) blockchains.MUMBAI = blockchains.MATIC
```
Comment (samholmes) [inline src/controllers/action-queue/WyreClient.js:90]: This doesn't really have any impact?

File: src/controllers/action-queue/WyreClient.js [hunk 3]
```diff
@@ -83,10 +87,10 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
...
         if (blockchains.TESTBTC == null) {
           const address = blockchains.BTC
-          if (['m', 'n', '2'].includes(address[0])) blockchains.TESTBTC = address
+          if (address != null && ['m', 'n', '2'].includes(address[0])) blockchains.TESTBTC = address
```
Comment (samholmes) [inline src/controllers/action-queue/WyreClient.js:93]: Ah, this makes sense however.

**Review by samholmes — CHANGES_REQUESTED** [review]
Looks like some cleanup needed to be done at the end. Also rebase onto develop now that the dependency PR is merged.

File: src/controllers/action-queue/WyreClient.js (hunk 2 above)
Comment (Jon-edge) [reply src/controllers/action-queue/WyreClient.js:90]: Good catch, this was misguided. The real issue was the test BTC. Still getting used to being allowed to reference nonexistent keys 😂

File: src/controllers/action-queue/WyreClient.js (hunk 1 above)
Comment (Jon-edge) [reply src/controllers/action-queue/WyreClient.js:45]: Good catch, missed this when debugging for my fixes.

File: src/controllers/action-queue/WyreClient.js (hunk 2 above)
Comment (Jon-edge) [reply src/controllers/action-queue/WyreClient.js:90]: Actually I think still has merit. Don't want to create a MUMBAI key at all if there's no MATIC configured.

Thinking on this a bit more, I don't think there's even a point to setting up MUMBAI at all to a live net address. Plus, MUMBAI really does not behave as reliably as live net

Comment (Jon-edge) [reply src/controllers/action-queue/WyreClient.js:90]: Going to just leave this for now. We do have a lot of Object.keys usage throughout the gui for map logic.

Comment (Jon-edge) [reply src/controllers/action-queue/WyreClient.js:90]: I think still has merit. Don't want to even create a MUMBAI key at all if there's no MATIC configured.

Thinking on this a bit more, I don't think there's even a point to setting up MUMBAI at all to a live net address. Plus, MUMBAI really does not behave as reliably as live net

Going to just leave this for now. We do have a lot of Object.keys usage throughout the gui for map logic.

**Comment by Jon-edge** [comment]
Will rebase after these changes are approved

**Review by samholmes — APPROVED** [review]
Approved but squash/apply my fixups commits

**Pull request merged**